lib/src/useTernaryDarkMode/useTernaryDarkMode.ts

```diff
@@ -9,56 +9,63 @@ import { useUpdateEffect } from '../useUpdateEffect'
 
 const COLOR_SCHEME_QUERY = '(prefers-color-scheme: dark)'
 
-enum TernaryDarkMode {
-  System = 'system',
-  Dark = 'dark',
-  Light = 'light',
-}
+type TernaryDarkMode = 'system' | 'dark' | 'light'
 interface UseTernaryDarkModeOutput {
   isDarkMode: boolean
   ternaryDarkMode: TernaryDarkMode
+  dispatchTernaryDarkMode: (ternaryDarkMode?: TernaryDarkMode) => void
```

Comment:
Here, there is a good practice, we can return the `dispatchTernaryDarkMode` with the same API of `setState` from `useState`.

```js
import { Dispatch, SetStateAction } from 'react'
```

```suggestion
dispatchTernaryDarkMode: Dispatch<SetStateAction<TernaryDarkMode>>
```

Notes: 
The `setTernaryDarkMode` from the `useLocalStorage` uses the same technique and thus has the same signature.